### ISSUE TITLE
Incorrectly sets a non-zero exit code when using opam template files.

### ISSUE THREAD

**Issue by tmcgilchrist** [ocurrent/opam-dune-lint#43]
If I have both an `gitlab-jsoo.opam` and `gitlab-jsoo.opam.template` in some situations the linting will trigger a warning. In particular running ` opam exec -- opam-dune-lint` without a `dune build @all` will trigger this warning. See https://ci.ocamllabs.io/github/tmcgilchrist/ocaml-gitlab/commit/db97416eed4c1cbd5673178c2eddc19f5208a0d9/variant/(lint-opam)

```shell
opam exec -- opam-dune-lint
gitlab-jsoo.opam: changed after 'dune build @install'!
gitlab-unix.opam: changed after 'dune build @install'!
gitlab.opam: changed after 'dune build @install'!
gitlab-jsoo.opam: OK 
gitlab-unix.opam: OK
gitlab.opam: OK
lab.opam: OK
*[tests][~/code/ocaml/ocaml-gitlab]$ echo $?                    
1
```

**Comment by talex5** [comment]
This is correct. It means you forgot to check in the generated opam files.
